h11/_headers.py

```diff
@@ -74,6 +74,11 @@ def normalize_and_validate(headers):
         # downstream." -- https://tools.ietf.org/html/rfc7230#section-3.2.4
         if name.strip() != name:
             raise LocalProtocolError("Illegal header name {}".format(name))
+        # "A sender MUST NOT generate a message that includes line folding"
+        # -- https://tools.ietf.org/html/rfc7230#section-3.2.4
```

Comment:
Hrmm, while I'm sure the other language is there in the RFC, I am struggling to find it at the moment. I went back and checked the tickets for when we did this in Requests and didn't find a specific reference there either.

Do you or @Lukasa know off hand which section the verbage is in, so we can reference it correctly? Otherwise, I'll take a deeper look tonight.